### DISCUSSION TITLE
Tracker issue for buildbot

### DISCUSSION THREAD

**Discussion by ghost** [Bforartists/Bforartists#2413] [General]
Should we add an issue for buildbot task?

**Comment by ReinerBforartists** [comment]
The issue already exists. The docker container is working, except one little detail. It is just that naetherm has simply no time at the moment.

https://github.com/Bforartists/Bforartists/issues/2284

**Reply by ghost** [reply]
Ah ok yep aware if this one, maybe we can rename it but anyways.
Btw what is the issue, maybe we can work on it together

**Reply by ghost** [reply]
have working recently on a buildbot cloud server
![image](https://user-images.githubusercontent.com/25552173/117778213-b0916580-b23d-11eb-855c-3697f7a01cd5.png)

**Reply by ReinerBforartists** [reply]
The issue is time ^^

He is working hard. Sometimes he has a few moments left, sometimes not. We are under no big pressure here. It would be fine to have now, no question. But the docker is working except a little thing. And it would not make sense to repeat this work now.

**Reply by ghost** [reply]
> The issue is time ^^
> 
> 
> 
> He is working hard. Sometimes he has a few moments left, sometimes not. We are under no big pressure here. It would be fine to have now, no question. But the docker is working except a little thing. And it would not make sense to repeat this work now.

Great then, agree ^^

**Comment by ReinerBforartists** [comment]
Thanks for your patience. When you eagerly search for work in this area, we have github actions, and it fails since months ^^

https://github.com/Bforartists/Bforartists/actions

**Comment by ghost** [comment]
When the docker is ready I think we can use buildbot or github actions, it would be great

**Comment by ReinerBforartists** [comment]
Github actions is a own system. Not connected.

**Reply by ghost** [reply]
Oh ok
